qcodes/instrument/delegate/delegate_instrument.py

```diff
@@ -95,24 +99,32 @@ def __init__(
         self,
         name: str,
         station: Station,
-        parameters: Union[Mapping[str, Sequence[str]], Mapping[str, str]],
+        parameters: Optional[Union[Mapping[str, Sequence[str]], Mapping[str, str]]] = None,
+        channels: Optional[Union[Mapping[str, Mapping[str, Any]], Mapping[str, str]]] = None,
         initial_values: Optional[Mapping[str, Any]] = None,
         set_initial_values_on_load: bool = False,
         setters: Optional[Mapping[str, MutableMapping[str, Any]]] = None,
         units: Optional[Mapping[str, str]] = None,
         metadata: Optional[Mapping[Any, Any]] = None,
     ):
         super().__init__(name=name, metadata=metadata)
-        self._create_and_add_parameters(
-            station=station,
-            parameters=parameters,
-            setters=setters or {},
-            units=units or {}
-        )
+        if parameters is not None:
+            self._create_and_add_parameters(
+                station=station,
+                parameters=parameters,
+                setters=setters or {},
+                units=units or {}
+            )
         self._initial_values = initial_values or {}
         if set_initial_values_on_load:
             self.set_initial_values()
```

Comment:
I guess the order of this code implies that initial_values only work for parameters and not for parameters on the channels? I don't know if that is an issue? Perhaps we should at least document that

Reply:
Or maybe change the order so that it works on channel parameters.